File: app/src/main/java/com/cappielloantonio/play/subsonic/models/ArtistsID3.java
package com.cappielloantonio.play.subsonic.models;

import com.tickaroo.tikxml.annotation.Element;
import com.tickaroo.tikxml.annotation.Xml;

import java.util.ArrayList;
import java.util.List;

@Xml
public class ArtistsID3 {
    @Element(name = "index")
    protected List<IndexID3> indices;
    protected String ignoredArticles;

    public List<IndexID3> getIndices() {
        if (indices == null) {
            indices = new ArrayList<>();
        }
        return this.indices;
    }

    public void setIndices(List<IndexID3> indices) {
        this.indices = indices;
    }

    public String getIgnoredArticles() {
        return ignoredArticles;
    }

    public void setIgnoredArticles(String value) {
        this.ignoredArticles = value;
    }
}

File: app/src/main/java/com/cappielloantonio/tempo/subsonic/models/ArtistsID3.kt
package com.cappielloantonio.tempo.subsonic.models

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName

@Keep
class ArtistsID3 {
    @SerializedName("index")
    var indices: List<IndexID3>? = null
    var ignoredArticles: String? = null
}